reporthandling/results/v1/reportsummary/controlsummarymethods.go

```diff
@@ -55,7 +55,8 @@ func (controlSummary *ControlSummary) Append(status apis.IStatus, ids ...string)
 		controlSummary.ResourceIDs.Append(status.Status(), ids[i])
 		controlSummary.increase(status)
 	}
-	controlSummary.ResourceIDs.ToUniqueResources()
+	// FIXME - Matthias thinks this one is useless
```

Comment:
@dwertent I think this one was just to prevent lists from growing too much... which is irrelevant with sets

Reply:
@matthyx  I concur. If remember correctly, there are other "uniqueness" demands in the reportshandling/results package